packages/dd-trace/test/encode/0.4.spec.js

```diff
@@ -1,6 +1,6 @@
 'use strict'
 
-require('../setup/tap')
+// require('../setup/tap')
```

Comment:
uh ?

Reply:
I commented the line in local to see the test execution integrated in my IDE... it works better if I ignore tap locally. But this shit sometimes happens 🤦 
Changed back again